### DISCUSSION TITLE
BMS abschaltung wegen Unterspannung -> Zwangs- / Notladung über Huawei?

### DISCUSSION THREAD

**Discussion by Seili76** [hoylabs/OpenDTU-OnBattery#720] [Q&A] [answered]
Heute Nacht ist es unerklärlicherweise bei mir dazu gekommen, das mein BMS den Akku wegen Unterspannung deaktiviert hat.
Ich kann aufgrund der Aufzeichnungen nicht feststellen, das die DTU hier der Auslöser war, es wurde augenscheinlich nicht über den WR eingespeist.

SOC Grenzen beim WR liegen bei 15%/25%, das hat bis dato auch immer gut geklappt.

Ob der Standby-Verbrauch des WR ausgereicht hat, um die Abschaltung auszulösen, ist für mich nicht nachzuvollziehen.
Die BMS Abschalt-Grenzwerte sind mir nicht bekannt, aber schon beim Hersteller angefragt. Letzter Eintrag im Batteriestatus war 45,41V, seitdem ist das BMS nicht mehr erreichbar.

Nun muss ich (nach Recherche) den Akku mit Ladespannung und kleinem Strom belegen, damit dieser wieder aus seinem Tiefschlaf erweckt werden kann. (BMS ist nicht erreichbar)

Ist das über Umwege momentan schon zu realisieren, obwohl derzeit bei uns kein PV Überschuss zu erwarten ist?

Ansonsten hätte ich einen Funktionswunsch:
Um das unabhängig von der PV Leistung machen zu können, wäre es prima, wenn man das Huawei über die DTU "zwangsweise" mit niedriger Leistung auch mit Netzbezug dazu veranlassen könnte, den Akku zu laden.

Als Ladespannung kann das voreingestellte Ladespannungslimit verwendet werden, der Strom sollte für die Funktion begrenzbar sein.

Da die Abschaltung der Akkus sicherlich nicht der Regelfall sein sollte, könnte eine solche Funktion auch genutzt werden um den Akku bei längeren ertragslosen Phasen nicht mit zu niedrigem SOC stehen lassen zu müssen, da der WR ja unweigerlich an der Batterie nuckelt.

Ist sowas realisierbar?

Ich hoffe das heute noch Ertrag vom Dach kommt, damit ich den Akku zumindest mit Ladespannung belegen kann...

Gruß
Markus

**Comment by hoschiking** [comment]
Dein Problem ist ganz einfach gelöst.
Der SOC ist völlig aus dem Ruder gelaufen, ist normal und nicht so einfach zu lösen.
Daher ist es seit neuestem möglich, für die Regelung der Einspeisung den SOC zu ignorieren und Schwellwerte für die Batteriespannung festzulegen. Nutze einfach dies, schon rennst du nicht gegen die BMS-Abschaltspannung.
Sinnvolle Werte sind: 51,2 V und 53,6V. 45,4V lässt auf einen komplett entladenen Speicher schliessen, damit meine ich ganz leer.
Um den Speicher aufzuwecken, wäre es am einfachsten ein Labornetzteil an den Eingang zu hängen, wenn verfügbar.

**Reply by Seili76** [reply]
Leider keines welches die angeforderten 54V liefert... Daher der Weg über das Huawei...

Den Weg über die Spannungen werde ich testen...

**Reply by hoschiking** [reply]
2 in Reihe, wenn 2 Netzteile hast....

**Reply by Seili76** [reply]
Leider nur ein regelbares mit 24V vorhanden... Ertrag steigt aber gerade, evtl. reicht es doch noch um 
den Huawei in Gang zu bekommen....

**Reply by hoschiking** [reply]
Ja, die nächsten Tage wird es genug Sonne geben. BTW: Bei mehreren Netzteilen in Reihe reicht es, wenn nur eines eine Strombegrenzung hat, die anderen müssen nicht mal geregelt sein

**Reply by Seili76** [reply]
Ich hatte jetzt kurz Überschuss, so das der Huawei kurz geladen hat. Dadurch ist zumindest das BMS wieder erreichbar.
Interessantes Verhalten des Akkus...

**Reply by schlimmchen** [reply, marked as answer]
Siehe #723.
